packages/typescript-tests/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "private": true,
+  "name": "typescript-tests",
+  "version": "0.14.1",
+  "type": "module",
+  "scripts": {
+    "typescript": "tsc"
```

Comment:
This script could be called `test` and then it will automatically run without needing changes to the root scripts or github workflow

Reply:
Good idea - though about this as well when implementing, so sounds reasonable to do it this way